pbm/restore/logical.go

```diff
@@ -79,13 +80,13 @@ type oplogRange struct {
 type restoreUsersAndRolesOption bool
 
 // New creates a new restore object
-func New(
-	leadConn connect.Client,
+func New(leadConn connect.Client,
```

Comment:
```suggestion
func New(
	leadConn connect.Client,
```
keep consistency in style